src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/DiagnosticsClient.cs

```diff
@@ -202,12 +207,27 @@ internal ProcessInfo GetProcessInfo()
                     var hr = BitConverter.ToInt32(response.Payload, 0);
                     throw new ServerErrorException($"Get process info failed (HRESULT: 0x{hr:X8})");
                 case DiagnosticsServerResponseId.OK:
-                    return ProcessInfo.Parse(response.Payload);
+                    return ProcessInfo.ParseV1(response.Payload);
                 default:
                     throw new ServerErrorException($"Get process info failed - server responded with unknown command");
             }
         }
 
+        private bool TryGetProcessInfo2(out ProcessInfo processInfo)
```

Comment:
Will you also be updating async versions of these once your other PR is checked in?

Reply:
I think this change will be merged long before the async + cancellation change does. Once this on is merged, I'll do a small refactor with this method to allow it to be async as well.